test/e2e/tests/tokens/custom-token-send-transfer.spec.ts

```diff
@@ -36,8 +31,8 @@ describe('Transfer custom tokens', function () {
           title: this.test?.fullTitle(),
           testSpecificMock: mocks,
         },
-        async ({ driver }) => {
-          await logInWithBalanceValidation(driver);
+        async ({ driver, localNodes }) => {
+          await loginWithBalanceValidation(driver, localNodes[0]);
```

Comment:
:information_source:  we pass the node because in these specs we deployed a smart contract, so our account balance won't be the 25 ETH, rather less, so we rely on the RPC call to get its value